ros_buildfarm/templates/snippet/builder_system-groovy_verify-upstream.xml.em

```diff
@@ -1,3 +1,6 @@
+@{
+code = TEMPLATE('snippet/check_recursive_upstream_projects.groovy.em')
+}@
```

Comment:
Should this block be inside the Groovy snippet? This seems to be created malconfigured builds.

I saw this first on my ROS 2 test farm but some recent jobs on build.ros.org like [this one](http://build.ros.org/view/Lbin_uxv8_uXv8/job/Lbin_uxv8_uXv8__libphidget21__ubuntu_xenial_arm64__binary/) which are currently failing due to script approval also look like something isn't quite right.